core/dbt/events/types.proto

```diff
@@ -15,6 +15,7 @@ message EventInfo {
     string thread = 7;
     google.protobuf.Timestamp ts = 8;
     map<string, string> extra = 9;
+    string category = 10;
```

Comment:
Is this eventually going to be the log category?  Am I missing how this gets populated?

Reply:
Yeah, right now it's just a placeholder. Eventually we'll populate it somehow...

Reply:
Is there an issue so we don't lose the fact we still need to actually populate it?

Reply:
Mostly it's your ticket #5958.